cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -288,10 +324,43 @@ data LiveViewState a = LiveViewState
     , lvsUIThread            :: Maybe (Async.Async ())
     , lvsMetricsThread       :: Maybe (Async.Async ())
     , lvsNodeThread          :: Maybe (Async.Async ())
+    , lvsNodeKernel          :: Maybe (NodeKernel IO ConnectionId blk)
+    , lvsPeers               :: [PeerUI blk]
```

Comment:
As I mentioned, I would create separation from this `NodeKernel` mess here and just introduce the variables that you actually use in the LiveView.
Coupling everything with references to huge data structures is a no-no in my book.
That's just my view.

Reply:
From my standpoint there is another important question -- what is the intent/mandate of the `LiveViewState` data structure?

Is it for keeping all of the state that the `LiveView` needs?  So far it has been.

Reply:
So we don't depend on the data structure that we don't use.
Why don't we have a single `GlobalState` passing around and be done with it?

Reply:
> So we don't depend on the data structure that we don't use.

But we _do_ depend on it!

Reply:
I'm not sure if there is some language barrier here or I'm not making sense but I'm not sure how you don't see that this question about `GlobalState` is exactly what you are describing above.

What  would be wrong if you had a field inside `LiveViewState`:
```
ppPeer :: Text
```

You would have to move the computation before assigning it to the data type and you wouldn't have to jump through all these hoops?

Reply:
@ksaric, how would that be computed?

Reply:
Replied in DM on Slack.
If what I wrote doesn't make sense, it's fine this way.
The costs/benefits all mentioned in this conversation.